BACKPORT.md

```diff
@@ -0,0 +1,39 @@
+- [Managing Backports in OpenSearch Clients](#managing-backports-in-opensearch-clients)
```

Comment:
Feels like duplicate information across repos, what do you think of pointing to https://github.com/opensearch-project/opensearch-plugins/blob/main/WORKFLOWS.md#managing-backports ?

Reply:
Sure! Linked to the plugins documentation.